src/PhpSpreadsheet/Spreadsheet.php

```diff
@@ -660,9 +660,10 @@ public function getAllSheets(): array
      */
     public function getSheetByName(string $worksheetName): ?Worksheet
     {
-        $worksheetCount = count($this->workSheetCollection);
-        for ($i = 0; $i < $worksheetCount; ++$i) {
-            if (strcasecmp($this->workSheetCollection[$i]->getTitle(), trim($worksheetName, "'")) === 0) {
+        $trimWorksheetName = trim($worksheetName, "'");
+
+        for ($i = 0; $i < $this->getSheetCount(); ++$i) {
+            if (strcasecmp($this->workSheetCollection[$i]->getTitle(), $trimWorksheetName) === 0) {
```

Comment:
This one, however, does improve things a bit by moving a function call from inside the loop to outside. I would certainly be okay with including this part.